src/components/Nav.js

```diff
@@ -1,71 +1,39 @@
 import React, { Component } from 'react';
-import AppBar from 'material-ui/AppBar';
-import Drawer from 'material-ui/Drawer';
-import MenuItem from 'material-ui/MenuItem';
 import { Link } from 'react-router-dom';
 
 class Nav extends Component {
-  constructor(props) {
-    super(props);
 
-    this.state = {
-      drawerOpen: false,
-    };
-
-    this.handleDrawerToggle = this.handleDrawerToggle.bind(this);
-  }
-  handleDrawerToggle() {
-    this.setState({
-      drawerOpen: !this.state.drawerOpen
-    });
-  }
   render () {
     return (
       <div>
-        <AppBar
-          title={
-            <Link style={{
-              color: '#ffffff',
-              textDecoration: 'none'
-            }} to="/">
-              Adopt-a-Family
+        <div>
+          <div className='header-text'>
+            <Link to='/'>
```

Comment:
Were we passing class names directly in to `<Link />` components to save lines on extra divs?